docker/domserver/build.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/sh -eu
 
+# Use venv to use latest Sphinx. 6.1.0 or higher is required to build DOMjudge docs.
+# shellcheck source=/dev/null
+. /venv/bin/activate
```

Comment:
Does it work without this one?

Reply:
We need this because `make install-docs` (at the last line of this file) runs `sphinx-build` command.

Reply:
So can you move it there, so its clear that we need it there? Alternative is to put it in the profile for all users so if people debug the container it would be activated automatically (just a suggestion, I leave this up to you if you think that's an improvement)

Reply:
Thanks for the suggestion! I've moved it there.

> to put it in the profile for all users so if people debug the container it would be activated automatically

This is interesting. To make this work, I had to specify `SHELL ["/bin/bash", "-l", "-c"]` to use login shell in Dockerfile, and add `-l` option to `sudo -u domjudge` commands in `docker/domserver/build.sh`. (See https://github.com/tossy310/domjudge-packaging/commit/8a00483fb9ccc81a2673c0bae4a603c8a48ee44b)
I have a mixed feeling that this looks simple and helps debugging, while using venv is a bit hacky and my initial approach is a bit more noticeable.

Currently in this PR I just moved `activate` just before it's needed. But let me know if profile approach seems better. I'm happy to update this PR.

Reply:
No fine with this, I'll merge it. Thank you for your work.